appholder/build.gradle

```diff
@@ -0,0 +1,82 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'androidx.navigation.safeargs.kotlin'
+    id 'kotlin-android-extensions'
+    id 'kotlin-kapt'
+}
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        applicationId "com.ul.ims.gmdl.appholder"
```

Comment:
Please use `com.android.mdl.app` as the application name (and `com.android.mdl` as the package name for all sources).

Reply:
Changed